### PR TITLE
CodeMirror 5.23.0

### PR DESCRIPTION
**Pull request by okonomiyaki3000**
Pull Request for Issue # .

### Summary of Changes
CodeMirror updated to latest version: http://codemirror.net/doc/releases.html

### Testing Instructions
Use CodeMirror, expect it to work properly.

### Expected result
CodeMirror works properly.

### Actual result
CodeMirror works properly.

### Documentation Changes Required
Nope.

### CODE REVIEW

**Comment by okonomiyaki3000** [comment]
What?

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on b72875c3cf0cdba3157a49b32d38f10a37450126<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13958">issues.joomla.org/tracker/joomla-cms/13958</a>.</sub>

File: plugins/editors/codemirror/codemirror.xml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <extension version="3.2" type="plugin" group="editors" method="upgrade">
 	<name>plg_editors_codemirror</name>
+<<<<<<< HEAD
```
Comment (C-Lodder) [inline plugins/editors/codemirror/codemirror.xml]: Please fix the conflicts

Comment (zero-24) [inline plugins/editors/codemirror/codemirror.xml]: fixed wirth: https://github.com/joomla/joomla-cms/pull/13958/commits/922cf49d6ebeddf2aaaf2320af45b8418b6a52de @okonomiyaki3000  @C-Lodder

**Comment by okonomiyaki3000** [comment]
@C-Lodder Thanks!

I don't get this AppVeyor failure. I think it has nothing to do with these changes. Is the staging branch currently broken?

**Comment by mbabker** [comment]
https://github.com/joomla/joomla-cms/pull/13941 apparently fixes it and a previously uncovered issue with our cache handling on Windows.  Don't worry about it.

**Comment by JoshuaLewis** [comment]
I have tested this item :white_check_mark: successfully on 37ece28eb680c3bb8ada2256dd24417cba2029ed<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13958">issues.joomla.org/tracker/joomla-cms/13958</a>.</sub>

**Comment by JoshuaLewis** [comment]
@franz-wohlkoenig Do you mind testing again? The Joomla tracker only has my successful test as seen here:
https://issues.joomla.org/tracker/joomla-cms/13958

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 37ece28eb680c3bb8ada2256dd24417cba2029ed<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13958">issues.joomla.org/tracker/joomla-cms/13958</a>.</sub>

**Comment by zero-24** [comment]
Thanks :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13958">issues.joomla.org/tracker/joomla-cms/13958</a>.</sub>

**Pull request merged**